upgrades.js

```diff
@@ -75,7 +75,7 @@ H5PUpgrades['H5P.InteractiveBook'] = (function () {
           };
 
           if (bookCover.coverAltText || bookCover.coverImage) {
-            bookCover.coverMedia = convertToImageParams(bookCover.coverImage, bookCover.coverAltText);
+            bookCover.coverMedium = convertToImageParams(bookCover.coverImage, bookCover.coverAltText);
```

Comment:
Why 🥺 If we want to change to cover media we should also change variable names I changed coverMedium to coverMedia everywhere 😓

Reply:
The reason is this whole code is for the 1.6 version and we are already in the 1.7 version so this code will not execute at all.